scripts/gh_scripts/weekly_status_report.mjs

```diff
@@ -213,7 +213,7 @@ async function prepareUntriagedIssues(leads) {
 
     let allIssuesTriaged = true
     for (const lead of leads) {
-        const searchResultsUrl = `https://github.com/internetarchive/openlibrary/issues?q=is%3Aissue+is%3Aopen+label%3A%22Needs%3A+Triage%22${encodeURIComponent('"' + lead.leadLabel + '"')}`
+        const searchResultsUrl = `https://github.com/internetarchive/openlibrary/issues?q=is%3Aissue+is%3Aopen+label%3A%22Needs%3A+Triage%22+label%3A${encodeURIComponent('"' + lead.leadLabel + '"')}`
```

Comment:
This is a great place for URLSearchParams; that'll also make this code easier to read, and less likely to have similar bugs in the future :+1: This is the pattern I usually use:

```suggestion
        const searchResultsUrl = `https://github.com/internetarchive/openlibrary/issues?${new URLSearchParams({
            q: `is:issue is:open label:"Needs: Triage" label:"${leadl.leadLabel}"`
        })}`
```

Reply:
This is def easier for me to grok, if those block works as is, I'd love to accept it and merge and we can verify everything works either by kicking off the task manually or wait until Friday. The cost is relatively low to fix if it doesn't work